cmd/network-console-collector/config.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"encoding/json"
+	"os"
+	"time"
+)
+
+type Config struct {
```

Comment:
note: Notice I've nixed the "console-auth" option for now. On openshift oauth still works mostly like it did before (assuming it is configured with the ouath-proxy.)

The only material loss here is basic auth. I'd entertain adding basic back in as an option if our hands are tied, but would want to rethink how it gets configured and implemented. Leaving it to the user to limit access by exposing the console in some restricted network space or using some ingress with custom authentication (like openshift oauth-proxy) is my preference.

Reply:
I agree, we you can do better than Auth basic, but that it's probably not worth spending too much effort on at the current status. 
The problem, however, is that the reason why we use a minimum of protection is not for the console but to protect the APIs and not those of the collector which honestly do not seem to me to contain sensitive data but those of Prometheus. We have had a problem in the past with QEs because we were publicly exposing the prometheus route = expose sensitive data. So I think they will raise the same issue if we don't protect the bees for a situation without OCP auth proxy

Reply:
Instead if we want to think about a solution maybe we keep in stanby keykloack but it would be another bomb to maintain and manage. Also I don't think is the direction we want to follow. Howver, just to mentionate something: Keycloak is currently used in Skupper X by Ted to grant ACLs based on potential user permissions (At the end probably will be discarded this solution for something more light)